assets/javascripts/discourse/templates/group-reports-index.hbs

```diff
@@ -0,0 +1,25 @@
+<section class='user-content'>
+  <table class='group-reports'>
+    <thead>
+      <th>
+        {{i18n "explorer.report_name"}}
+      </th>
+      <th>
+        {{i18n "explorer.query_description"}}
+      </th>
+      <th>
+        {{i18n "explorer.query_time"}}
+      </th>
+    </thead>
+    <tr></tr>
+    <tbody>
+      {{#each queries as |query|}}
+        <tr>
+          <td><a href="reports/{{query.id}}">{{query.name}}</a></td>
```

Comment:
Can you use `{{link-to}}` here? If not this link will need to be aware of subfolder installations of Discourse. (usually a call to `Discourse.getURL`

Reply:
We can!